tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -5792,6 +5792,43 @@ def _impl(ctx):
         bitcode_embedded_markers_feature = feature(name = "bitcode_embedded_markers")
         bitcode_embedded_feature = feature(name = "bitcode_embedded")
 
+    if (ctx.attr.cpu == "armeabi-v7a" or
+        ctx.attr.cpu == "darwin_x86_64" or
+        ctx.attr.cpu == "ios_arm64" or
+        ctx.attr.cpu == "ios_arm64e" or
+        ctx.attr.cpu == "ios_armv7" or
+        ctx.attr.cpu == "ios_i386" or
+        ctx.attr.cpu == "ios_x86_64" or
+        ctx.attr.cpu == "tvos_arm64" or
+        ctx.attr.cpu == "tvos_x86_64" or
+        ctx.attr.cpu == "watchos_arm64_32" or
+        ctx.attr.cpu == "watchos_armv7k" or
+        ctx.attr.cpu == "watchos_i386" or
+        ctx.attr.cpu == "watchos_x86_64"):
```

Comment:
it looks like this is all cpus? so maybe it shouldn't be in an if?

Reply:
I wasn’t sure about this. Would it be safe to assume that future added cpus support this feature?

Reply:
I think so

Reply:
there are other features not wrapped in ifs, so that seems safe

Reply:
It's really more a function of the linker. Unless Apple ships a new CPU that requires a different linker, it should be safe. 

If they do, then you'll have to update more than this stanza.

Reply:
Thanks. Updated.